apps/web-giddh/src/app/shared/header/header.component.ts

```diff
@@ -215,7 +215,12 @@ export class HeaderComponent implements OnInit, AfterViewInit, OnDestroy, AfterV
     public commonLocaleData: any = {};
     /** This holds the active locale */
     public activeLocale: string = "";
-
+    /** True if sidebar is expanded */
+    public isSidebarExpanded: boolean = false;
+    /** This will hold if setting icon is disabled */
+    public isSettingsIconDisabled: boolean = false;
+    /* This will hold if resolution is more than 767 to consider as ipad screen */
```

Comment:
Incorrect doc: `isIpadScreen` is set on 768px but in doc 767 is mentioned
`this.isIpadScreen = result?.breakpoints['(max-width: 768px)'];`

Reply:
Done